app/models/miq_ae_instance.rb

```diff
@@ -132,12 +132,21 @@ def self.copy(options)
     end
   end
 
-  def self.get_homonymic_across_domains(user, fqname, enabled = nil)
+  def self.display_name(number = 1)
+    n_('Automate Instance', 'Automate Instances', number)
+  end
+
+  def self.get_homonymic_across_domains(user, fqname, enabled = true, prefix: true)
```

Comment:
@pkomanek Why did the `enabled` default change from `nil` to `true`.   I would not expect that value to change.

Reply:
@gmcculloug my fault. Fixed.